feelpp/feel/feelmesh/meshsupport.hpp

```diff
@@ -134,7 +134,9 @@ public :
                 return M_rangeMeshElementsGhostIdsPartialSupport.find( eltId ) != M_rangeMeshElementsGhostIdsPartialSupport.end();
         }
 
-    bool isGhostFace( face_type const& face ) const
+    template <typename FaceType>
+    bool isGhostFace( FaceType const& face,
+                      typename std::enable_if_t<std::is_same_v<FaceType,face_type> >* = nullptr ) const
         {
             if ( M_isFullSupport )
                 return face.isGhostFace();
```

Comment:
use if constexpr instead of SFINAE. it simplifies the code

Reply:
I think it complexifies a little bit the reading here

Reply:
Why? Less lines and clearer than sfinae.

Reply:
we read only the code associated with the method, the other case is only for compilation issue, we never go here normally. For me it's here a little bit simpler otherwise we have two if-else nested.